africanlii/views/authors.py

```diff
@@ -43,8 +43,20 @@ def get_context_data(self, **kwargs):
             | Q(judgment__author=author)
         )
 
-        years = list(set(docs.values_list("date__year", flat=True)))
-        doc_types = list(set(docs.values_list("doc_type", flat=True)))
+        years = list(
```

Comment:
Do you need to re-calculate years? The `FilteredDocumentListView` class already does it, I think, so it's already in `context`?